consensus/src/block_storage/block_store.rs

```diff
@@ -262,6 +269,9 @@ impl BlockStore {
             .expect("Failed to persist commit");
 
         self.inner.write().update_ordered_root(block_to_commit.id());
+        self.inner
+            .write()
+            .insert_ordered_cert(finality_proof_clone.clone());
```

Comment:
i'd combine this function with `update_ordered_root` to take another arg

Reply:
Is there a need to distinguish between highest_ordered_cert and highest_ordered_root at all?
Can't we always derive the highest_ordered_root from highest_ordered_cert?